usaspending_api/api_docs/markdown/documentation_index.md

```diff
@@ -8,7 +8,9 @@
 # Documentation Index <a name="documentation-index"></a>
 
 ## Tutorials <a name="tutorials"></a>
-* [USASpending API Introductory Tutorial](/docs/intro-tutorial) - A short tutorial covering basic API usage, generally covering: endpoints, filtering, and autocompletes.
+- [How to USe USAspending API Endpoints](https://www.youtube.com/watch?v=AEKL2LOkRZY) - A 10-minute video discussing how to use the USAspending API with couple of examples.
+  O [Side deck](https://www.usaspending.gov/data/Basic-API-Training.pdf) - with more detailed information about how to use USAspending API endpoints.
+- [Introductory Tutorial](https://api.usaspending.gov/docs/intro-tutorial) -  short tutorial covering basic API usage, generally covering: endpoints, filtering, and autocompletes.
```

Comment:
Should be `A short tutorial` to be more consistent with the first bullet.

Reply:
done